pkg/cmd/kafka/list/list.go

```diff
@@ -127,18 +123,13 @@ func runList(opts *options) error {
 	}
 
 	switch opts.outputFormat {
-	case dump.JSONFormat:
-		data, _ := json.Marshal(response)
-		_ = dump.JSON(opts.IO.Out, data)
-	case dump.YAMLFormat, dump.YMLFormat:
-		data, _ := yaml.Marshal(response)
-		_ = dump.YAML(opts.IO.Out, data)
-	default:
+	case "table":
```

Comment:
I would define this as constat in dump: https://github.com/redhat-developer/app-services-cli/blob/roles/pkg/dump/dump.go#L23:L23

Reply:
Sure, give me a sec.

Reply:
Done

Reply:
The last time I checked, "table" is	even a possible value for `--output`. Has it been added?

Reply:
table should be default

Reply:
"table" is not currently a valid output format. Currently an empty string maps to the output table format.

https://github.com/redhat-developer/app-services-cli/blob/e34a650f24a6f291ec375a936f9431060f9ce5b6/pkg/cmdutil/flags/flags.go#L10

Which I think is fine, because there will be no circumstance where `--output=table` is ever needed.

Reply:
Yes, I've mentioned that in this comment https://github.com/redhat-developer/app-services-cli/pull/1066#discussion_r708358320. Do you want me to change TableFormat to an empty string?